### DISCUSSION TITLE
Enhancement — For Joomla Article Astroid Banner option to have settings for padding and text alignment

### DISCUSSION THREAD

**Discussion by Chacapamac** [templaza/astroid-framework#283] [Q&A]
I think it will be cool to have some settings for the Astroid Article Banner option for padding and text alignment

**Comment by sonvnn** [comment]
Hi @Chacapamac 

Thanks for your suggestion. May you detail your idea? I am working on a new template for Astroid. I will try to test this idea on that template.

Best Regards,
Sonny

**Comment by Chacapamac** [comment]
Ideally I will see a device specific control of the banner, at least for the padding and height.
**As other idea for the banner...**
I will also make a footer exactly the same way to the content as it is sometime use in some design.

**One question:**
**Why Device Specific options in Astroid present only 3 choice when bootstrap have 6 breakpoints?**
 I know its to simplify, but sometime you need the full control of bootstrap to make a perfect design. 

**IDEA; 
Maybe having an “Expert” mode that rely on all Bootstrap Breakpoints?**
Like that, older template continue to work and people are presented with the actual simplified choice and for the people that want more control they can click an “expert” button to reveal the full Bootstrap breakpoints....
